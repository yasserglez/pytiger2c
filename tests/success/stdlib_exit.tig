for i:= 0 to 10 do(
    if i > 4 then exit(1);
    print("this should be printed 5 times\n")
)
