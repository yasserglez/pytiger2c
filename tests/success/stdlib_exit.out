this should be printed 5 times
this should be printed 5 times
this should be printed 5 times
this should be printed 5 times
this should be printed 5 times
